applications/CableNetApplication/tests/cable_net_test_factory.py

```diff
@@ -0,0 +1,106 @@
+from __future__ import print_function, absolute_import, division
+import KratosMultiphysics as KM
+
+import KratosMultiphysics.KratosUnittest as KratosUnittest
+import KratosMultiphysics.kratos_utilities as kratos_utils
+
+
+from KratosMultiphysics.CoSimulationApplication.co_simulation_analysis import CoSimulationAnalysis
+from KratosMultiphysics.StructuralMechanicsApplication.structural_mechanics_analysis import StructuralMechanicsAnalysis
+
+
+import KratosMultiphysics.CableNetApplication
```

Comment:
better add a comment why this is needed

Reply:
I thought there was an error msg relating to an element which was not found in case I don't import this app ....
just tried without the import and it still works, so I'll remove this line

Reply:
probably bcs the app was already imported by another test